go.mod

```diff
@@ -61,12 +62,16 @@ require (
 	github.com/fatih/color v1.13.0 // indirect
 	github.com/fsnotify/fsnotify v1.6.0 // indirect
 	github.com/fxamacker/cbor/v2 v2.4.0 // indirect
-	github.com/go-git/gcfg v1.5.0 // indirect
-	github.com/go-git/go-billy/v5 v5.4.0 // indirect
-	github.com/go-git/go-git/v5 v5.5.2 // indirect
+	github.com/getkin/kin-openapi v0.107.0 // indirect
```

Comment:
This is one of the several new dependencies that `go mod tidy` explicitly adds to go.sum now that we oapi-codegen is an explicit dependency via tools.go. Before that, they were only used implicitly, by being included in the precompiled oapi-codegen. I guess it's good for dependency awareness that we're tracking them more explicitly now.